tutorials/containers/kubeflow-tutorial/kubeflow-tutorial.md

```diff
@@ -115,33 +185,68 @@ ks pkg install kubeflow/tf-serving
 ks pkg install kubeflow/tf-job
 ```
 
+positive
+: **ksonnet rate limit**
+If you hit a "403 API rate limit of 60 still exceeded" error you will need to register with Github. Please follow the [rate limit workaround](https://github.com/ksonnet/ksonnet/blob/master/docs/troubleshooting.md#github-rate-limiting-errors).
+
 ksonnet will pick up the configuration in our local `~/.kube/config` and prepare the environment for us. We can then proceed to apply the parameters to the prototypes and deploy the Kubeflow core components (JupyterHub and the TensorFlow job controller):
 
 ```bash
 ks generate core kubeflow-core --name=kubeflow-core --namespace=kf-tutorial
+```
+
+CDK users should enter the following:
+
+```bash
 ks apply cdk -c kubeflow-core
 ```
 
-You should see some informational messages confirming the deployment. Let's look at the Kubernetes Dashboard to verify. It should look similar to this (note the namespace "kf-tutorial"):
+As GKE has RBAC enabled and our user has insufficient permissions, we need to grant the *admin* role to our user with an additional command before running the same *apply* command as CDK users:
```

Comment:
wouldn't it be more logical to put this conditional step before the other one then?

Reply:
great suggestion!